File: practicas/practica3-ed/arrayExample.js
//en JS el array es dinamico
const numbers = [10,12,21,33,14,35,56,77,88,99]
const arrSize = numbers.length
console.log(`The size of the array is ${arrSize}`)

console.log(numbers[0])
console.log(numbers[9])
console.log(numbers[99])

numbers[0] = 101
numbers[11] = 101

console.log(numbers)

console.log("Recorriendo arreglo")
for(let i = 0; i < arrSize; i++){
    console.log(numbers[i])
}

File: practicas/practica3-ed/arrayExample.cpp
// Basci Array Example
//To compile g++ arrayExample.cpp -o arrayExample

#include <iostream>
int main (int argv, char **argc){
    int numbers[10] = {10,12,21,33,14,15,56,77,88,99};

    int arrSize = sizeof(numbers)/sizeof(numbers[0]);
    std::cout << "The size of the array is: " << arrSize << std::endl;

    numbers[9] = 25;

    std::cout << numbers[0] << std::endl;
    std::cout << numbers[9] << std::endl;

     std::cout << "Recorriendo el arreglo " << std::endl;
     for (int i = 0; i < arrSize; i++){
         std::cout << numbers[i] << std::endl;
     }

    return 0;
}